app/frontend/src/store/modules/user.js

```diff
@@ -30,44 +31,58 @@ export default {
           acr.prodStatus = false;
         });
       }
-    }
+    },
+    setModuleLoaded: (state, status) => state.moduleLoaded = status
   },
   actions: {
     /**
-     * @function getUserAcronyms
-     * Fetch the acronyms the current user has access to from the DB
+     * @function getAcronymClientStatus
+     * Fetch the service clients for the acronyms the user has
      * @param {object} context The store context
      */
-    getUserAcronyms(context) {
-      if (Vue.prototype.$keycloak &&
-        Vue.prototype.$keycloak.ready &&
-        Vue.prototype.$keycloak.authenticated) {
-        UserService.getUserAcronyms(Vue.prototype.$keycloak.subject)
-          .then(response => {
-            if (response && response.data) {
-              context.commit('setAcronyms', response.data);
-            }
-          })
-          .catch(error => {
-            // TODO: Create top-level global state error message
-            console.error(error); // eslint-disable-line no-console
-          });
+    async getAcronymClientStatus(context) {
+      try {
+        if (context.rootGetters['auth/authenticated']) {
```

Comment:
I don't like this.  Is there any way to bury that in the service/class or some kind of abstraction?  Is  context.rootGetters['auth/subject'] used in more places than this?  I imagine context.rootGetters['auth/authenticated'] will be. Are we going to wrap all calls to services with that?

Reply:
This is how vuex stores communicate with each other, I'm not sure there's a way around it. Maybe could extract to a utility method in the same store. If possible that would be better because hard coding the string isn't ideal if it's used more than once.
So this wouldn't be used in components, services, etc. Only in vuex modules themselves when they need to talk to a parent or sibling module

Reply:
@jujaga look at the section "Accessing Global Assets in Namespaced Modules" here
https://vuex.vuejs.org/guide/modules.html
might be able to make the syntax a bit cleaner at least?